pkg/daemon/ceph/client/mirror.go

```diff
@@ -168,3 +168,26 @@ func GetPoolMirroringInfo(context *clusterd.Context, clusterInfo *ClusterInfo, p
 
 	return &poolMirroringInfo, nil
 }
+
+// EnableSnapshotSchedule configures the snapshots schedule on a mirrored pool
+func EnableSnapshotSchedule(context *clusterd.Context, clusterInfo *ClusterInfo, poolSpec cephv1.PoolSpec, poolName string) error {
+	logger.Infof("enabling snapshot scheduling for pool %q", poolName)
+
+	// Build command
+	args := []string{"mirror", "snapshot", "schedule", "--pool", poolName, poolSpec.Mirroring.SnapshotSchedule.Interval}
+
+	// If a start time is defined let's add it
+	if poolSpec.Mirroring.SnapshotSchedule.StartTime != "" {
+		args = append(args, poolSpec.Mirroring.SnapshotSchedule.StartTime)
+	}
+	cmd := NewRBDCommand(context, clusterInfo, args)
+
+	// Run command
+	buf, err := cmd.Run()
```

Comment:
Is this cmd idempotent, or do we need to check for an error code if it already exists in a later reconcile?

Reply:
@dillaman is the `rbd mirror snapshot schedule`  command idempotent? Thanks

Reply:
They are idempotent (for now) but there has been a request to add some feedback (i.e. "add" returning EEXISTS and "remove" returning ENOENT).